cymetric/metrics.py

```diff
@@ -280,6 +280,10 @@ def transaction_quantity(series):
 ## FCO-related metrics ##
 #########################
 
+# Some useful variables
+x_feed = 0.0072
+x_tails = 0.0025
```

Comment:
I don't have much of a feel for what should/shouldn't be global. I can keep them within the metric for now and if it needs to be global I'll adhere to the style guide better in that change